tests/thread_msg_avail/Makefile

```diff
@@ -0,0 +1,8 @@
+APPLICATION = thread_msg_avail
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_MEMORY := stm32f0discovery
```

Comment:
Are you sure? This application just uses `core`. If `stm32f0discovery` does not fit `core` we really reevaluate support for this platform ;-)